fe/src/main/java/org/apache/doris/catalog/Catalog.java

```diff
@@ -3908,6 +3907,12 @@ public static void getDdlStmt(Table table, List<String> createTableStmt, List<St
                 sb.append(olapTable.getTableProperty().getDynamicPartitionProperty().toString());
             }
 
+            // 7. replicationNum
```

Comment:
Do not add replication num in `create table stmt`, cause it does not reflect the real replication num of each partition of a table.

Reply:
I think just like the DISTRIBUTED BY HASH(id) BUCKETS 5, which also does not reflect the real hash info of each partition of a table. so in the create table it just show the baisc config for the table and user can still set different distribution info for the special partition

Reply:
OK, I agree with u.